pinodo/coffee-mathcer
# feat: Set up Eslint for code formatting

**Summary**

Set up Eslint for code formatting. 
References Issue #9 

**Steps to test**
- Open `App.jsx` and make changes to the code format. For example:
  - Replace the single quotes with double quotes
  - Add trailing spaces
  - Remove the blank line at the end of the file
- Run `npm run lint` to view linting errors
- Run `npm run lint -- --fix` to automatically fix the errors
- Run `npm run lint` again to ensure the errors no longer persist
